Formula/distcc.rb

```diff
@@ -14,8 +13,26 @@ class Distcc < Formula
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  depends_on "python3" => :build
+
+  resource "libiberty" do
+    url "http://http.debian.net/debian/pool/main/libi/libiberty/libiberty_20180425.orig.tar.xz"
```

Comment:
Can we use something like `https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/libi/libiberty/libiberty_20180425.orig.tar.xz` instead of a plaintext link?

Reply:
Good idea, done!